### PR TITLE
Russian translations for interface

### PR DESCRIPTION
**Pull request by fizvlad**
Added Russian translation

(Issue #219)

### CODE REVIEW

**Comment by fizvlad** [comment]
@hovancik

File: app/locales/ru.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    "monitorIdleTime": "считать бездействие за естественный перерыв",
+    "showOnAllScreens": "отображать на всех экранах",
+    "language": "язык",
+    "resetToDefaults": "настройки по умолчанию"
+  },
+  "utils": {
+    "secondsLeft": "{{count}} секунда осталась",
+    "secondsLeft_plural": "{{count}} секунд осталось",
+    "minutesLeft": "{{count}} минута осталась",
+    "minutesLeft_plural": "{{count}} минут осталось",
```
Comment (hovancik) [inline app/locales/ru.json]: With plurals, you should have 3 possibilities for Russian language: 
key_0, key_1, key_2, eg: minutesLeft_0, minutesLeft_1, minutesLeft_2

You can read more here: 
https://www.i18next.com/plurals.html#how-to-find-the-correct-plural-suffix

Let me know if you need help with this

**Comment by fizvlad** [comment]
@hovancik 
Hey! I added keys "key_0", "key_1", "key_2" (Suggested suffixes for i18next v3) instead of "key" and "key_plural".
Is this right?

**Comment by hovancik** [comment]
Yes, what is your platform? I can send you file to test if you haven't run it locally to test. You should be able to run it locally thought ;]

**Comment by fizvlad** [comment]
I'm using Linux Mint (amd_64) and use *_amd64.deb file to install.

Also
screenshot
Is this cool? Seems like missing _'\n'_ at the beginng of _main.pausedIndefinitely_

**Comment by hovancik** [comment]
You've just found issue ;] Are you up to fix it?

**Comment by fizvlad** [comment]
Sure

**Pull request merged**